pkg/console/starter/starter.go

```diff
@@ -103,13 +106,35 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
-	recorder := ctx.EventRecorder
+	resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		api.OpenshiftConfigManagedNamespace,
+	)
 
 	operatorClient := &operatorclient.OperatorClient{
 		Informers: operatorConfigInformers,
 		Client:    operatorConfigClient.OperatorV1(),
 	}
 
+	resourceSyncer := resourcesynccontroller.NewResourceSyncController(
+		operatorClient,
+		resourceSyncerInformers,
+		v1helpers.CachedSecretGetter(kubeClient.CoreV1(), resourceSyncerInformers),
+		v1helpers.CachedConfigMapGetter(kubeClient.CoreV1(), resourceSyncerInformers),
+		ctx.EventRecorder,
+	)
+
+	// We need to sync the console-config configmap from openshift-config-managed because it may contain necessary branding configuration.
+	if err := resourceSyncer.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: api.OpenShiftConsoleOperatorNamespace, Name: api.OpenshiftManagedConfigMap},
+		resourcesynccontroller.ResourceLocation{Namespace: api.OpenshiftConfigManagedNamespace, Name: api.OpenShiftConsoleConfigMapName},
+	); err != nil {
+		logrus.Errorf("failed to copy managed configmap: %v \n", err)
```

Comment:
If it's a `404`, will this error always display?   Because it may not actually be an error, correct?  We are ok with a `not found`.

Reply:
The SyncConfigMap function there is only checking for the namespaces at that point and is not checking if there is a specific file.  An error will only be thrown if the namespaces do not exist.  We check later in the code when we are looking for the specific file for a 404.